File: May-18/26-05-18/java/01-Ques-2.java

import java.util.*;

class TestClass {
    public static void main(String args[] ) throws Exception {
        Scanner input = new Scanner(System.in);
        int n, x, y;
        n = input.nextInt();
        x = input.nextInt();
        y = input.nextInt();
        boolean flag = true;
        for(int i = 0; i < n; i++)
        {
            int val = input.nextInt();
            if(val < x || val > y)
                flag = false;
        }
        if(flag == true)
            System.out.println("YES");
        else
            System.out.println("NO");
        
        
    }
}

File: May-18/26-05-18/c++/01-Ques-2.cpp
#include<bits/stdc++.h>

using namespace std;

int main()
{
    int n, x, y;
    cin >> n >> x >> y;
    bool flag = true;
    
    for(int i = 0; i < n; i++)
    {
        int val;
        cin >> val;
        if(val < x or val > y)
            flag = false;
    }
    if(flag)
        cout << "YES\n";
    else
        cout << "NO\n";
}
